packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -1,14 +1,14 @@
-import React from "react";
+import { SelectClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { classNames, KeyCodes } from "@microsoft/fast-web-utilities";
+import { canUseDOM } from "exenv-es6";
 import { get, isEqual, isNil } from "lodash-es";
```

Comment:
Looks like line 243 is still using `get`

Reply:
it is - I think this will get fixed with the fix from @scomea